content/blog/lift-and-transform-apps-with-hpe-csi-driver-for-kubernetes.md

```diff
@@ -0,0 +1,283 @@
+---
+title: Lift and Transform Apps with HPE CSI Driver for Kubernetes
+date: 2022-06-15T18:17:50.974Z
+author: Michael Mattsson
+authorimage: /img/portrait-192.png
+tags:
+  - kubernetes
+  - hpe-alletra
+  - hpe-nimble-storage
+  - hpe-3par-and-primera
+---
+![]()
+
+Do you run your applications in virtual machines on VMware today using the same patterns once employed by bare-metal paradigms? Just a bit more efficient from a resource consumption perspective, but what about the lifecycle of the application, its dependencies - including libraries, host operating systems and various mnemonics to keep the application up and running?
+
+Containerization and Kubernetes in particular help businesses to move away from imperative operational models which require multiple teams to perform tedious tasks on a regular basis towards a declarative model where there’s a clean separation of concerns with a high degree of automation and abstractions that makes sense for running a high-performance technology company. 
+
+In this blog post we’ll discuss a methodology that can be employed for lifting and transforming legacy stateful applications running on VMware vSphere to Kubernetes using HPE Alletra, Nimble Storage or Primera and the HPE CSI Driver for Kubernetes. The destination Kubernetes cluster could potentially be running on VMware vSphere or any other hypervisor, but it’s not a requirement, as long as the Kubernetes cluster and the origin VMware vSphere environment has connectivity to the underlying array, we’re in good shape.
+
+The TL;DR version of this blog post is available as an instructional lightboard video [published on YouTube](https://www.youtube.com/watch?v=M7t_qPe3i5E). The blog concretizes what’s being outlined in the lightboard video. 
+
+[![YouTube](/img/untitled.jpg)](https://www.youtube.com/watch?v=M7t_qPe3i5E)
+
+Still want the details? Let’s get started!
+
+# Origin state
+
+In the following examples we’re going to use a MariaDB database with the blessed [Employees database](https://github.com/datacharmer/test_db) as an example workload. It provides simple means to validate its contents to ensure the contents are intact throughout its journey.
```

Comment:
Place a comma after "In the following examples".
Suggest taking out "blessed", as we've shown no reason previous that it should be anything otherwise. I could see using it if you mentioned earlier that there had been some trouble figuring out which database to use, but since we haven't, it's not needed and might cause folks to wonder what you mean.

In the second sentence, place "a" between "provides" and "simple"; replace "to" with "and"; and add "that" prior to "the contents" so it reads:

It provides a simple means to validate its contents and ensure that the contents are intact throughout its journey.